contents/blog/why-redesign.mdx

```diff
@@ -0,0 +1,171 @@
+
+### 2. New navigation menu grouping
+
+<div class="sm:max-w-sm sm:float-right -mt-12 -mb-12 sm:-ml-8">
+
+![PostHog 3000 nav menu](../images/blog/why-redesign/nav.png)
+
+</div>
+
+The nav menu has now been organized into two main sections: **project & data** and **products**.
+
+**Project & data** includes your dashboards and notebooks, along with the data you're sent to PostHog (like events, people, and properties), and metadata you've added on top (like actions, cohorts, and annotations) – and a live feed of incoming events.
+
+**Products** is the ever-growing list of tools built on top of the PostHog platform.
+
+Once the icons are familiar enough, you may want to hide the text labels. You can do this by dragging the dividing line to the left of the screen (or just double-click it).
+
+In the future, we may add the [ability to customize the nav menu](https://github.com/PostHog/posthog/issues/18884) by re-ordering items or hiding products you don't use.
+
+<div class="clear-both">&nbsp;</div>
+
+### 3. Side panel
+
+The new collapsible side panel is always available on the right side of the screen and is designed to help with whatever you're currently doing.
+
+- **Notebooks:** Drag almost anything from the main workspace into a notebook so you don't lose it. It's designed to be a scratchpad for data exploration.
+- **In-app docs (seen below):** Clicking any docs link in the product will now open in the side panel, keeping you in the flow of what you're doing.
+- **Support:** Get in touch with support without having a model blocking whatever it is you were looking at.
+
+![PostHog 3000 in dark mode](../images/blog/why-redesign/side-panel.gif)
+
+The side panel may also open other panes as necessary – like quick access to settings, or an onboarding checklist if you're new to PostHog.
+
+### 4. Notebooks
+
+[Notebooks](#) is a large enough feature that it deserves a post of its own, but in short, it's a way to load up multiple insights (and more) onto a single page. As you explore data, keep dragging in more findings until you come to a conclusion. Then share it with your team.
+
+### 5. Toolbar 3000
+
+It's the same toolbar you know and love, except now it doesn't look like it was designed in 2020! (Oh, and it also has dark mode.)
+
+![Toolbar](../images/blog/why-redesign/toolbar.jpg)
+
+### 6. Command bar
+
+Press `Cmd` + `Shift` + `K` to launch the new command bar. Navigate to most of PostHog's built-in features without ever touching your trackpad. (This includes things like generating an API key, creating a dashboard, or switching to dark mode.)
+
+![Command bar](../images/blog/why-redesign/command-bar.jpg)
+
+### 7. Reorganized settings
+
+Clicking on _project settings_ will now take you to a more organized page of settings.
+
+---
+
+## So what's next?
+
+This reskin is really just the beginning. There's no shortage of ideas of what to work on next. Here are some other experience improvement’s we’re considering:
```

Comment:
As a user "There's no shortage of ideas" makes me a bit scared of what will change in the future 😅 I think we should drop it, as the list itself is much better at conveying how we concretely want to improve the product.